polkadot-parachains/integritee-runtime/src/lib.rs

```diff
@@ -219,8 +287,8 @@ parameter_types! {
 
 impl pallet_timestamp::Config for Runtime {
 	/// A timestamp: milliseconds since the unix epoch.
-	type Moment = u64;
-	type OnTimestampSet = ();
+	type Moment = Moment;
+	type OnTimestampSet = Teerex; // aura needed? https://github.com/integritee-network/integritee-node/blob/master/runtime/src/lib.rs#L316
```

Comment:
Solochain also includes aura here. Statemint however does not. What's the way go here?

Reply:
Quite sure, we do it like Statemint.